README.md

```diff
@@ -1,3 +1,14 @@
+**PREVIEW OF CHANGES**
+- Searching movies from the search box
+- Pagination if search result gives > 20 movies
+- Sorting functionality from select box, on average vote or on popularity
+- Add movie to the cart from the red cart button on the movie component
+- Add/Remove movie from cart buttons
+- Calculating a total price from a fixed price (api doesnt provide price) and show it in the cart
+- Total movies in the cart appear on the header
+- Showing messages for no results in movie search and empy cart
+
+
```

Comment:
Very nice job Maria! 
In general, you are doing great! I would love to see also the implementation of removing a move from the title of the movie. Like when you add a movie to the cart, not to add more but to change the add button to a remove button!

Reply:
Keep an eye on your console, try to resolve warnings!